src/Tizen.NUI/src/internal/WebView/WebBackForwardList.cs

```diff
@@ -58,15 +63,55 @@ public string Title
         /// Get original url.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Uri OriginalUrl
+        public string OriginalUrl
         {
             get
             {
-                return new Uri(Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr));
+                return Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr);
             }
         }
     }
 
+    /// <summary>
+    /// WebBackForwardCopiedListItems is a class for back-forward copied list item of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardCopiedItemList : Disposable
+    {
+        internal WebBackForwardCopiedItemList(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        {
+            Interop.WebBackForwardCopiedItemList.DeleteCopiedItems(swigCPtr);
+        }
+
+        /// <summary>
+        /// Get item count.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint ItemCount
+        {
+            get
+            {
+                return Interop.WebBackForwardCopiedItemList.GetItemCount(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Get item with index.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebBackForwardListItem GetItemAtIndex(uint index)
```

Comment:
Please add a description of the parameter.
```C#
/// <summary>
/// ...
/// </summary>
/// <param name="index">... </param>
```
Please update other APIs as well

Reply:
fixed